content/en/logs/guide/azure-logging-guide.md

```diff
@@ -0,0 +1,275 @@
+    a. Event Parameter Name is set to `eventHubMessages`.  
+    b. Event Hub Cardinality is set to `Many`.  
+    c. Event Hub Data Type is left empty.  
+8. Click **Save**.
+9. Verify your setup is correct by running the function and then checking the [Datadog log explorer][6] for the test message.  
+**Note**: The test log event must be in valid JSON format.
+
+#### Activity logs
+
+1. In the Azure portal, navigate to the **Activity Log**.
+2. Click on **Diagnostic Settings**.
+3. Click **Add diagnostic setting**.
+4. Under category details, select the categories of logs you want to send to Datadog.
+5. Under destination details, select **Stream to an event hub**.
+6. Set the Event Hub namespace and name. These should match the Event Hub namespace and name that you used to create your Event Hub trigger.
+7. Set the shared access key. This key should be configured with send or manage access.
+8. Click **Save**.
+9. Verify your setup is correct by checking the [Datadog log explorer][6] for logs from this resource.
+
+#### Resource logs
+
+Configure your Azure services to forward their logs to the Event Hub by creating a [diagnostic setting][3].
+
+1. In the Azure portal, navigate to the resource of the logs you want to send to Datadog.
+2. Under the monitoring section of the resource blade, click **Diagnostic settings**.
+3. Click **Add diagnostic setting**.
+4. Under category details, select the categories of logs you want to send to Datadog.
+5. Under destination details, select **Stream to an event hub**.
+6. Set the Event Hub namespace and name. These should match the Event Hub namespace and name that you used to create your Event Hub trigger.
+7. Set the shared access key. This key should be configured with send or manage access.
+8. Click **Save**.
+9. Verify your setup is correct by checking the [Datadog log explorer][6] for logs from this resource.
+
+[1]: https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create
+[2]: https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-event-hubs-trigger
+[3]: https://docs.microsoft.com/en-us/azure/azure-monitor/platform/diagnostic-settings
+[4]: https://github.com/DataDog/datadog-serverless-functions/blob/master/azure/activity_logs_monitoring/index.js
+[5]: https://app.datadoghq.com/organization-settings/api-keys
+[6]: https://app.datadoghq.com/logs
+[7]: https://docs.datadoghq.com/getting_started/site/
+
+{{% /tab %}}
+
+{{% tab "Blob Storage" %}}
+
+To collect logs from all of your Azure App Services, follow this general process:
+
+1. Set up [Azure Blob Storage][1] from the [Azure portal][2], [Azure Storage Explorer][3], [Azure CLI][4], or [Powershell][5].
+2. Set up the [Datadog-Azure Function](#create-a-new-azure-blob-storage-function) which forwards logs from your blob storage to Datadog.
+3. Configure your Azure App Services to [forward their logs to the Blob Storage][6].
+
+#### Create a new Azure Blob Storage function
```

Comment:
IMO we should add a note that this is not the suggested option for log forwarding to make it clear that we want customers to use eventhubs instead

Reply:
Thanks for clarifying that -- I added a note to this effect, but let me know if you have any feedback on it.